content/devdocs/references/p2p_networking.md

```diff
@@ -191,7 +260,29 @@ The following annotated hexdump shows an `ANNB` message with two
 inventory entries.  (The message header has been omitted.)
 
 {{ <highlight text> }}
-TODO: ANNB message hex dump
+AnnounceBlocksMessage:
+```
+0c 00 00 00 00 00 00 00 40 e2 01 00 00 00 00 00 
+................................................... Full message
+------------------------------------------------------------------------
+0x0000 0c 00 00 00 00 00 00 00 ........................... Prefix
```

Comment:
This is not quite like it should be . First four bytes are the total size of the buffer that follows . The other four bytes are the message `Prefix` ID (e.g.one of `INTR`, `GETB`, ...) . Please fix that